kin-sdk/kin-sdk-lib/src/test/kotlin/kin/sdk/internal/queue/PaymentQueueManagerTest.kt

```diff
@@ -0,0 +1,170 @@
+package kin.sdk.internal.queue
+
+import com.nhaarman.mockitokotlin2.*
+import kin.base.KeyPair
+import kin.sdk.internal.data.PendingBalanceUpdater
+import kin.sdk.internal.events.EventsManager
+import kin.sdk.queue.PendingPayment
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.junit.Before
+import org.junit.Test
+import java.math.BigDecimal
+
+class PaymentQueueManagerTest {
+
+    object Constants {
+        const val ONE_MILLI = 1000L
+        const val DELAY_BETWEEN_PAYMENTS_MILLIS = 3000L
+        const val QUEUE_TIMEOUT_MILLIS = 5000L
+        const val MAX_NUM_OF_PAYMENTS = 5
+    }
+
+    private var queueScheduler: FakeQueueScheduler = spy() //TODO use a spy in order to be able to verify method calls on that object, is this fine?
```

Comment:
no need spy for that, spy is if you want to invoke some of the original object code under some terms.

Reply:
I did it because I am using it with 'verify', for example, in 
`enqueue items, verify that schedule queue timeout happens only for first item`
method

Reply:
still, don't need it, verify is a mock feature as well, spy gives you `thenCallRealMethod`